opamp/config.go

```diff
@@ -57,3 +60,36 @@ func ParseConfig(configLocation string) (*Config, error) {
 
 	return &config, nil
 }
+
+// GetSecretKey returns secret key if set else returns empty string
+func (c Config) GetSecretKey() string {
+	if c.SecretKey == nil {
+		return ""
+	}
+
+	return *c.SecretKey
+}
+
+// CmpUpdatableFields compares updatable fields for equality
+func (c Config) CmpUpdatableFields(o Config) (equal bool) {
```

Comment:
extremely minor and i wouldn't hold up a pr for it, but is cmp a standard abbreviation for compare? maybe it is? all i know is i didn't know what this was doing until i looked at the code/read the description

Reply:
must just be my brain misfiring 😄